vector-im/element-android
# [Location Share] - Expanded map state when no more live location shares (PSG-629)

## Type of change


## Content


Adding a dedicated view when there are no more active lives when in the expanded map screen.

## Motivation and context


Closes #6635 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/180781807-609445e2-233d-489a-9dfc-517c4dbaba83.png" width=300 /> <img src="https://user-images.githubusercontent.com/46314705/180781819-2eefbc61-847f-4836-a1a6-1a5141932efa.png" width=300 />

## Tests



- Start a live location share in a room
- Go to the expanded map view when pressing the corresponding tile on the timeline
- Check the users list of active lives is correctly displayed
- Stop the live
- Check the users list dissapears
- Check a banner saying the live is ended is displayed a the bottom

## Tested devices

- OS version(s): Android 11

## Checklist
